force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -294,20 +355,45 @@ public with sharing class ClassesObjectsMethods {
      */
     public class Person {
         // Declare private two instance variables - (TEXT) name and (Number) age
-        
+        private String name;
+        private Integer age;
         // Constructor to initialize the name and age
-    
+        public Person (String name, Integer age){
+            this.name = name;
+            this.age = age;
+        }
         // Method introduceYourself that returns the details of the person        
         // Return a string in the format "Hello, my name is <name> and I am <age> years old."
         // For example, introduceYourself() should return a string like "Hello, my name is John Doe and I am 28 years old."
 
+        
+        public string introduceYourself(){
+            return 'Hello, my name is ' +this.name +' and I am ' +this.age +' years old.';
+        }
         // Getter method that returns the name of the person
-
+        public String getName(){
+            return name;
+        }
         // Getter method that returns the age of the person
-
+        public Integer getAge(){
+            return age;
+        }
         // Setter method that sets the name of the person
-
+        //*I do not understand why I needed to add void to this method or if this is correct
```

Comment:
You add void since it's not returning anything (there's no return statement). You're simply assigning the instance variable (declared at the start of the class) a value.